instaloader/__main__.py

```diff
@@ -133,7 +143,7 @@ def _main(instaloader: Instaloader, targetlist: List[str],
           max_count: Optional[int] = None, post_filter_str: Optional[str] = None,
           storyitem_filter_str: Optional[str] = None,
           browser: Optional[str] = None,
-          cookiefile: Optional[str] = None) -> None:
+          cookiefile: Optional[str] = None) -> int:
```

Comment:
It seems we could be even more explicit regarding the typing here:

```suggestion
          cookiefile: Optional[str] = None) -> ExitCodes:
```